ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh

```diff
@@ -19,12 +19,18 @@ snyk_deps() {
     if [ "$ALL_PROJECTS" = "true" ]; then
         PARAMS+=(--all-projects)
     fi
+    if [ "$SNYK_DEPS_ADDITIONAL_ARGS" ]; then
+        PARAMS+=("$SNYK_DEPS_ADDITIONAL_ARGS")
```

Comment:
Actually we should split this. As written it'll only work correctly for a single arg.

/hold

Reply:
Done.

/hold cancel